src/plugins/timeline.ts

```diff
@@ -49,7 +49,7 @@ export type TimelinePluginEvents = BasePluginEvents & {
 }
 
 class TimelinePlugin extends BasePlugin<TimelinePluginEvents, TimelinePluginOptions> {
-  private timelineWrapper: HTMLElement
+  public timelineWrapper: HTMLElement
```

Comment:
Every time we make something internal like this public, it increases the support surface area and makes upgrading to newer versions harder for users.

E.g. `this.wavesurfer.getWrapper` was never meant to be used by anything other than internally by plugins, and it says so in the doc comment.

So I would suggest reverting this.

Reply:
Reverted 👍🏽